client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -1508,6 +1511,32 @@ private TxnSegments convert(CreateTxnResponse response) {
                 });
     }
 
+    @Override
+    public CompletableFuture<List<UUID>> listCompletedTxns(Stream stream) {
+        Exceptions.checkNotClosed(closed.get(), this);
+        Preconditions.checkNotNull(stream, "stream");
+        final long requestId = requestIdGenerator.get();
+        long traceId = LoggerHelpers.traceEnter(log, "listCompletedTxns", stream, requestId);
+
+        final CompletableFuture<ListCompletedTxnResponse> result = this.retryConfig.runAsync(() -> {
```

Comment:
can we rename the variable `result` with a meaningful name? ex:- `completedTxnsList`?

Reply:
Done as suggested.